web/css/palettes.css

```diff
@@ -135,14 +135,14 @@
 
 .wv-palettes-panel .wv-running-bar::before {
   top: -1px;
-  left: -5px;
+  left: -4px;
```

Comment:
This looks off to me now:
![Screen Shot 2021-08-09 at 3 30 18 PM](https://user-images.githubusercontent.com/1480000/128777159-4ce7dbcd-c731-427f-b647-9234029f5e7f.png)

Here's what it looked like before:
![Screen Shot 2021-08-09 at 3 30 47 PM](https://user-images.githubusercontent.com/1480000/128777160-4ea4da05-eb90-40e0-83e2-e172bfc03b02.png)

Reply:
Or perhaps it's actually due to the changes in `web/js/components/sidebar/paletteLegend.js`

Reply:
I made a few changes there that shifted it over. Just revised that part in latest to fix.